features/indexeddb.yml.dist

```diff
@@ -0,0 +1,378 @@
+# Generated from: indexeddb.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+status:
+  baseline: high
+  baseline_low_date: 2020-01-15
+  baseline_high_date: 2022-07-15
+  support:
+    chrome: "58"
+    chrome_android: "58"
+    edge: "79"
+    firefox: "51"
+    firefox_android: "51"
+    safari: "10.1"
+    safari_ios: "10.3"
+compat_features:
+  # baseline: high
+  # baseline_low_date: 2015-07-29
+  # baseline_high_date: 2018-01-29
+  # support:
+  #   chrome: "24"
+  #   chrome_android: "25"
+  #   edge: "12"
+  #   firefox: "16"
+  #   firefox_android: "22"
+  #   safari: "8"
+  #   safari_ios: "8"
+  - api.IDBCursor
+  - api.IDBCursor.advance
+  - api.IDBCursor.continue
+  - api.IDBCursor.delete
+  - api.IDBCursor.direction
+  - api.IDBCursor.key
+  - api.IDBCursor.primaryKey
+  - api.IDBCursor.source
+  - api.IDBCursor.update
+  - api.IDBCursorWithValue
+  - api.IDBCursorWithValue.value
+  - api.IDBDatabase
+  - api.IDBDatabase.close
+  - api.IDBDatabase.createObjectStore
+  - api.IDBDatabase.deleteObjectStore
+  - api.IDBDatabase.name
+  - api.IDBDatabase.objectStoreNames
+  - api.IDBDatabase.transaction
+  - api.IDBDatabase.version
+  - api.IDBDatabase.versionchange_event
+  - api.IDBFactory
+  - api.IDBFactory.cmp
+  - api.IDBFactory.deleteDatabase
+  - api.IDBFactory.open
```

Comment:
This is the weird one. I'm guessing that BCD should be saying that this feature is partially supported on iOS from 14.5 to 15 (given the set of releases BCD tracks). But then we'd break alignment with caniuse.

https://bugs.webkit.org/show_bug.cgi?id=226547